src/parse-mockdb.js

```diff
@@ -112,8 +112,7 @@ function runHook(className, hookType, data) {
       object: model,
       user: "ParseMockDB doesn't define request.user."
     };
-
-    return hook(beforeSaveOrBeforeDeleteRequestObject).bind(model)().then((beforeSaveOverrideValue) => {
+    return hook(beforeSaveOrBeforeDeleteRequestObject).then((beforeSaveOverrideValue) => {
```

Comment:
i wish i understood what the .bind(model)() was supposed to do and if it should be there.  but now the unit tests pass.

:(  any insight you guys can give me as to what it was supposed to do, ideally in the form of a failing test so i can try to figure it out.....